scripts/eslint/es5Config.js

```diff
@@ -0,0 +1,3 @@
+const baseConfig = require('./baseConfig');
+
+module.exports = baseConfig.config;
```

Comment:
I think there's more to it. I think we want to have `ecmaVersion` set to `6` for the ES6 config and to `5` for the ES5 config. It's not just about var, but about all the other features too.

Reply:
ok, I add it on new commit.